src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -1261,6 +1261,24 @@ private FacadeRenderState getFacadeRenderState( EnumFacing side )
 			IBlockState blockState = facade.getBlockState();
 			if( blockState != null && textureItem != null )
 			{
+				try
+				{
+					blockState = blockState.getActualState( world, pos );
+				}
+				catch( Exception e )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)